tests/integration/ef_tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+from utils import load_all_ef_blockchain_tests
+
+
+# This enables devs to use the familiar keyword (-k argument) to select a subset of the EF test suite,
+# which are loaded as fixtures.
+def pytest_generate_tests(metafunc):
+    if "ef_blockchain_test" in metafunc.fixturenames:
+        keyword = metafunc.config.getoption("keyword")
+
+        if not keyword:
+            # with no keyword, we parametrize as empty
+            metafunc.parametrize(
+                "ef_blockchain_test",
+                [],
+                ids=[],
+            )
+        else:
+            start_time = time.time()
+            ef_blockchain_tests = load_all_ef_blockchain_tests()
+            # Calculate and print the elapsed time
+            elapsed_time = time.time() - start_time
+            print(f"Time taken to load_ef_blockchain_tests: {elapsed_time} seconds")
+
+            ef_blockchain_test_ids, ef_blockchain_test_objects = (
```

Comment:
what's the point of creating a util if its output is not the final correct form?